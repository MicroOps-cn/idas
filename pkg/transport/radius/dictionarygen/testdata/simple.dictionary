ATTRIBUTE ShortAttr 128 short
