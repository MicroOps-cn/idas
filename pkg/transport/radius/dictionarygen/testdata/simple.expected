Constants:
	const (
		ShortAttr_Type radius.Type = 128
	)
Types:
	type ShortAttr uint16
		func (a ShortAttr) String() string
Functions:
		func ShortAttr_Add(p *radius.Packet, value ShortAttr) (err error)
		func ShortAttr_Del(p *radius.Packet)
		func ShortAttr_Set(p *radius.Packet, value ShortAttr) (err error)
